__tests__/app.test.js

```diff
@@ -117,6 +117,38 @@ describe('GET /api/articles/', () => {
 				});
 			});
 	});
+	test('200: return all articles with the specified topic', () => {
+		return request(app)
+			.get(`/api/articles/?topic=mitch`)
+			.expect(200)
+			.then(({ body }) => {
+				const { articles } = body;
+				expect(articles.length).not.toBe(0);
+				for (const article of articles) {
+					expect(article).not.toHaveProperty('body');
+					expect(article).toMatchObject({
+						article_id: expect.any(Number),
+						title: expect.any(String),
+						topic: 'mitch',
+						author: expect.any(String),
+						created_at: expect.any(String),
+						votes: expect.any(Number),
+						article_img_url: expect.any(String),
+					});
+				}
+				expect(articles).toBeSortedBy('created_at', {
+					descending: true,
```

Comment:
This is a well executed thorough test 👍 You could check the exact length of the array on line 126 to make your test slightly more robust